Sources/Plasma/FeatureLib/pfPython/pySceneObjectGlue.cpp

```diff
@@ -179,6 +179,15 @@ PYTHON_METHOD_DEFINITION_NOARGS(ptSceneobject, getPythonMods)
     return retVal;
 }
 
+PYTHON_METHOD_DEFINITION_NOARGS(ptSceneobject, getImageLibMods)
+{
+    std::vector<PyObject*> vecList = self->fThis->GetImageLibMods();
+    PyObject* retVal = PyList_New(vecList.size());
+    for (int curKey = 0; curKey < vecList.size(); curKey++)
+        PyList_SetItem(retVal, curKey, vecList[curKey]); // steals the vecList ref
```

Comment:
`List` indicates a sequence that's purposefully mutable.

Reply:
`ptSceneobject.getResponders` and `ptSceneobject.getPythonMods` both return PyLists as well.

Reply:
Those are also wrong 😛

Reply:
Okay.